mathext/ell_carlson.go

```diff
@@ -0,0 +1,145 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import (
+	"math"
+)
+
+// CarlsonRF computes the symmetric elliptic integral R_F(x,y,z):
```

Comment:
Why are these called `Carlson...`? Wouldn't it be clearer to call these by the name of the integral somehow? Maybe `SymEllipticRF`?